README.md

```diff
@@ -1,2 +1,91 @@
 # PyCon2019
-Quickstart and example code for the PyCon 2019 Circuit Playground Express, sponsored by Digi-Key and Adafruit!
+This repo contains the Circuit Python Quickstart worksheet and example code for the PyCon 2019 
+Circuit Playground Express, sponsored by Digi-Key and Adafruit!
+
+Everyone will be receiving a Circuit Playground Express in their swag bag! Getting started with it
+requires a micro USB to USB cable. We will have a few for use during the CircuitPython Open Spaces,
+however we do not have enough for everyone. **Please remember to bring a micro USB cable with you!**
+
+**The Adafruit Circuit Playground Express (CPX) has CircuitPython on board! It’s a MicroChip
+SAMD21 microcontroller running at 48 MHz, with 32kB RAM and 256kB flash: 192kB for CircuitPython,
+plus 64kB for the CIRCUITPY USB drive. It's loaded with all kinds of sensors, LEDs, touch pads,
+buttons and more!**
+
+#### Check out these Adafruit Learn Guide Links!
+* Welcome to CircuitPython: http://adafru.it/cpy-welcome
+* CPX Guide: https://adafru.it/adafruit-cpx
+* CP Made Easy on CPX: https://adafru.it/cp-made-easy-on-cpx 
+
+#### Download the latest CircuitPython for Circuit Playground Express!
+* CircuitPython for CPX: https://circuitpython.org/board/circuitplayground_express/
+
+#### Are you on Windows 7?
+You need to install drivers before plugging in! See [Welcome->Installing
+CircuitPython](https://learn.adafruit.com/welcome-to-circuitpython/installing-circuitpython). 
+
+Windows 10, Mac, and Linux don’t need drivers.
+
+#### Plug It In!
+Use a micro-USB cable with data (beware charge/power-only cables). A USB drive called **CIRCUITPY**
+will appear. If there’s a **code.py** on **CIRCUITPY**, it will run automatically. 
```

Comment:
It will be shipping as `code.py` and we want to encourage use of `code.py` over `main.py`, I'm not sure I agree with adding this into the quickstart documentation.

Reply:
OK by me, not a big deal.